packages/engine/Source/Core/ArcGISTiledElevationTerrainProvider.js

```diff
@@ -455,10 +455,14 @@ ArcGISTiledElevationTerrainProvider.prototype.requestTileGeometry = function (
 
         // Don't reject the promise till the request is actually cancelled
         // Otherwise it will think the request failed, but it didn't.
-        return request.deferred.promise.finally(function () {
-          request.state = RequestState.CANCELLED;
-          return Promise.reject(error);
-        });
+        try {
+          await request.deferred?.promise;
+        } catch {
+          // Eat this error
```

Comment:
What is the reason not bubbling this error up or at least sending it to console.error?

Silently suppressing the error can make it harder to debug when something does go wrong